src/gui/mainmenumanager.h

```diff
@@ -140,7 +140,7 @@ class MainGameCallback : public IGameCallback
 
 	virtual void changeVolume()
```

Comment:
name this

Reply:
good catch, I had forgotten about doing that.